### DISCUSSION TITLE
Is it possible to specify a load balancer algorithm when using the Stateful Session Filter?

### DISCUSSION THREAD

**Discussion by cybrhwk** [istio/istio#48880] [Q&A] [answered]
We are using the stateful session filter enabled via the pilot environment variable:

PILOT_ENABLE_PERSISTENT_SESSION_FILTER set to "1" for pilot

and applying the label to my service:

istio.io/persistent-session: somecookie

Is there a way to specify what load balancing algorithm is used for new sessions?

**Comment by ramaraochavali** [comment, marked as answer]
Set the Load balancing algorithm in Destination Rule. that will be used for new sessions
